src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -1158,3 +1163,11 @@ int sd_dhcp_server_set_ntp(sd_dhcp_server *server, const struct in_addr ntp[], u
 
         return 1;
 }
+
+int sd_dhcp_server_set_emit_router(sd_dhcp_server *server, uint8_t emit_router) {
```

Comment:
`uint8_t emit_router` should be `int enabled` instead. C.f. `sd_dhcp6_client_set_information_request`.